oas_docs/output/kibana.serverless.yaml

```diff
@@ -7749,6 +7713,12 @@ paths:
   /api/entity_store/status:
     get:
       operationId: GetEntityStoreStatus
+      parameters:
+        - description: If true returns a detailed status of the engine including all it's components
+          in: query
+          name: withComponents
```

Comment:
snake_case it?

Reply:
I agree, snake case > camel for yaml files. Or fit the style of the file already - which looks like snake case 🐍